### ISSUE TITLE
[Bug]: FC0006 - False Positive for AccessByPermission

### ISSUE THREAD

**Issue by pri-kise** [ALCops/Analyzers#474]
### Analyzer

FormattingCop

### Issue type

False positive (diagnostic fires when it shouldn't)

### Rule ID

FC0006 

### Description

The `AccessByPermission` property shouldn't trigger this info.

### AL code to reproduce

```al
                action("BOM Level")
                {
                    AccessByPermission = tabledata "BOM Buffer" = R;
                    ApplicationArea = Assembly;
                    Caption = 'BOM Level';
                    Image = BOMLevel;
                    ToolTip = 'View availability figures for items on bills of materials that show how many units of a parent item you can make based on the availability of child items.';

                    trigger OnAction()
                    begin
                        PurchAvailabilityMgt.ShowItemAvailabilityFromPurchLine(Rec, "Item Availability Type"::BOM)
                    end;
                }
```

### Additional context

https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/properties/devenv-accessbypermission-property?wt.mc_id=d365bc_inproduct_alextension

**Comment by Arthurvdv** [comment]
I've published a new beta release, `v1.2.0-beta.1`, which should resolve this issue.

Could you verify for me if it's fixed on your end? Would be great to hear whether it's resolved or if you're still running into problems.

If you're using the [ALCops VS Code extension](https://marketplace.visualstudio.com/items?itemName=Arthurvdv.alcops), you can switch to the beta channel in the extension settings. To force an update check, run the `ALCops: Check for Updates` command from the command palette.

**Comment by Arthurvdv** [comment]
This should now be resolved with the release of `v1.2.0` of ALCops.

When you have a chance, could you update and verify whether the issue is fixed?

If everything is working as expected, feel free to close this issue. If you still run into the problem, please leave the issue open and share any additional context, code samples, or reproduction steps that might help me investigate further.

**Comment by NKarolak** [comment]
Works for me now :-)
